### PR TITLE
Todo list code review

### PR DESCRIPTION
**Pull request by majovanilla**
Requesting code review for the Todo list project.

We followed the requirements in both The Odin Project and Microverse.

We'll Appreciate your comments!

### CODE REVIEW

File: src/js/controller/projectCtrl.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+function findProject(id) {
+  updateProjects();
+  const ids = projects.map(current => current.id);
+  const index = ids.indexOf(parseInt(id, 10));
+  return projects[index];
+}
+
+function validateInput(element) {
+  if (element.value === '') {
+    // element.classList.add('alert');
```
Comment (mcrd25) [inline src/js/controller/projectCtrl.js]: Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂

Comment (majovanilla) [reply src/js/controller/projectCtrl.js]: Changes made! Thanks for the review!

File: src/js/controller/projectCtrl.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-let projects = [];
+let projects = [{ title: 'Default project', id: 0, todoList: [] }];
```
Comment (majovanilla) [inline src/js/controller/projectCtrl.js]: Required change.

File: src/js/controller/projectCtrl.js [hunk 3]
```diff
@@ -51,12 +51,13 @@ function findProject(id) {
 
 function validateInput(element) {
   if (element.value === '') {
-    // element.classList.add('alert');
     return false;
   }
-  // element.classList.remove('alert');
   return true;
 }
```
Comment (majovanilla) [inline src/js/controller/projectCtrl.js]: Deleted commented code

**Pull request merged**